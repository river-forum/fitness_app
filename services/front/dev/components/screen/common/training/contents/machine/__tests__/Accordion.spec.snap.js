import { mount } from '@vue/test-utils'
import Accordion from '../Accordion.vue'

it('Accordion Component renders correctly', () => {
  const app = mount(Accordion)

  expect(app.html()).toMatchSnapshot()
})
