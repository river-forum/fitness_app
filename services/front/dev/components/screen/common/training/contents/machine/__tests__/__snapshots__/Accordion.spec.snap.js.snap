// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordion Component renders correctly 1`] = `
"<div class=\\"px-6 accordion\\"><button type=\\"button\\" class=\\"px-2 py-4 text-left accordion__trigger\\">

  </button>
  <transition-stub name=\\"js-accordion\\">
    <!---->
  </transition-stub>
</div>"
`;
